sponsors/index.html

```diff
@@ -42,7 +42,33 @@
     </script>
 
     <section id="sponsors">
-        <h1 align="center" class="content-header">Sponsors</h1>
+        <h1 align="center"  class="content-header">Sponsor Us!</h1>
+
+
+        <div align= "center" class="sponsor-call">
+            We are committed to cultivating diverse voices in our
+            student body while maintaining engagement between company representatives and our members in
+            whatever setting that may be. With your contribution, we will plan in-person and hybrid opportunities through
+            which we aim to help members gain confidence in pursuing their ambitions. In turn, you will be able to connect with these dedicated students and gain more recognition within our organization!
+        </div>
+
+        <div class="sponsor-call" style="padding-bottom: 0px;">
+          <p style="font-size: 20px;">Sponsorship Tiers:</p>  
+        </div>
+
+
+        <div align="center" class="sponsor-benefits color-indicator">
+            <span style="display:inline;" class="color-indicator platinum">PLATINUM</span>
+            <span style="display:inline;" class="color-indicator gold">GOLD</span>
+            <span style="display:inline;" class="color-indicator silver">SILVER </span>
+            <span style="display:inline;" class="color-indicator bronze">BRONZE</span>
+        </div>
+        
+        <div class="" style="text-align:center;padding-top:15px;padding-bottom:30px;">
+            <img src=/assets/img/logos/sponsors/sponsor_tiers.png style="height:50%;width:50%">
```

Comment:
on second thought, I think it might be better if we hid the sponsorship tiers behind a dropdown (similar to what y'all had before) since we probably don't want the tiers to be the first thing that members/students who are interested in wcs to see. lmk if you have alternative ideas!